content/news/2025-01-07-bevy-at-rust-week/index.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Bevy at RustWeek 2025: come hack with us!"
+date = 2025-01-07
+authors = ["Alice I. Cecile"]
+[extra]
+github = "alice-i-cecile"
++++
+
+[Contributing to Bevy](https://bevyengine.org/learn/contribute/introduction/) is a blast, but what if you could meet each other in flesh?
+
+[RustWeek 2025](https://rustweek.org/) has generously offered to let Bevy (among other Rust-affiliated groups!) nest into a room (plus a breakout room!) as part of their [Unconf](https://rustweek.org/unconf/). Three days of socializing, collaborating and working on Bevy, in whatever form we'd like!
+
+Now, for the details:
+
+- **Who:** Anyone who is actively contributing to Bevy or the surrounding Rust game dev ecosystem.
+  - If you're focused on Bevy's dependencies (`winit`, `wgpu` and so much more) please come hang out with us: we'd love to have you!
+  - Experienced users are welcome too: share your hopes, dreams, complaints and frustrations about Bevy
+  - If you have dietary restrictions or need accommodations of any form, please reach out to the RustWeek organizers and we'll do our best to meet your needs.
+- **What:** work on Bevy building and improving your favorite open source Rust game engine.
+  - What exactly will we work on? Who knows!
+  - There will be mad science.
+  - We will write docs.
+  - It *will* be fun.
+- **Where:** Utrecht, The Netherlands
+  - De Utrechtse Stadsvrijheid, to be exact.
+  - If you need help with letters of invitation or visas, let us know ASAP and we'll help get you sorted.
+- **When:** May 16th and 17th, 2025
+  - That's Friday and Saturday, with a bonus event on Thursday afternoon and evening.
+  - The conference beforehand is great (Alice is giving a talk!), but totally optional.
+  - Unconf attendees also invited to the 10 Years of Rust Celebration beforehand on May 15th!
+- **How much:** Free!
+  - You do need to pay for your own travel and lodging. We'll help coordinate hotel sharing for those who are interested.
+  - Renting a bike is recommended: the Netherlands is great fun with one and it was extremely useful for RustNL 2024. We'll try to coordinate a group deal!
+  - Bevy and RustWeek would like to offer some financial support to contributors who need it: email or DM us if this would make life easier for you <3
+
+If you'd like to attend, add yourself to [this spreadsheet](https://docs.google.com/spreadsheets/d/1-WwFHj2GdZ8quon7V5z8XNVQDDYvme7_-QzfBGelRBA/edit?usp=sharing). We have limited slots (about 20), and people need to make travel arrangements: please be considerate when indicating your level of committedness! If you later find you cannot attend, please remove yourself from this list so we can free up space for others.
```

Comment:
I think people are going to want to know when they are "confirmed" to be attending. Is it first-come-first-serve or will confirmations be sent out a specific point? What happens in the (perhaps unlikely) event that we get more interest than our given set of slots.